default-config.json

```diff
@@ -33,6 +33,17 @@
       "description": "Request a list of songs in the Cadence database"
     }
   },
+  "commandDetails": {
+    "help": {
+      "internalKey": "help",
+      "subtitle": "The help command provides information to assist with the usage of Cadence.",
+      "lines": [
+        "It can be used in one of two ways:",
+        "    1. Without any argument (`%help`), it will generate a summary of commands and a brief snippet of their purpose.",
```

Comment:
After my offline conversation with @JunhaoSLi, we agreed this would be a good syntax to insert the value of `config.commands.help` - IE the triggering phrase for the specified command. Here's how this looks:

![image](https://user-images.githubusercontent.com/15851480/103470391-72d47b80-4d37-11eb-8a7e-29cf3b97d983.png)